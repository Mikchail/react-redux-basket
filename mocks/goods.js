const goods = [
  {
    id: '2',
    img:
      'https://img.favpng.com/2/4/5/steel-guitar-acoustic-guitar-electric-guitar-bass-guitar-png-favpng-MYXXnrv6tC6Hg2T1k7zKjW8pr.jpg',
    name: 'Fender',
    price: 60000,
    quantity: 22,
  },
  {
    id: '22',
    img: 'https://img95.699pic.com/element/40135/1104.png_860.png',
    name: 'Fender',
    price: 22000,
    quantity: 2,
  },
  {
    id: '123',
    img:
      'https://img.favpng.com/20/11/20/bass-guitar-electric-guitar-acoustic-guitar-supro-ozark-1560-s-png-favpng-Auv04m1w16rLVMr6RZn433D7P_t.jpg',
    name: 'Fender',
    price: 14000,
    quantity: 22,
  },
  {
    id: '1212',
    img:
      'https://img.favpng.com/22/25/5/bass-guitar-art-string-instruments-drawing-png-favpng-NEmJB8UcuuuUiN1ipbpHEJYCF.jpg',
    name: 'Fender',
    price: 12000,
    quantity: 12,
  },
  {
    id: '2009',
    img:
      'https://img.pngio.com/download-electric-guitar-png-images-background-transparent-bass-hd-bass-guitar-png-840_767.png',
    name: 'Fender',
    price: 15000,
    quantity: 2,
  },
];

const basket = [
  {
    id: '9fbe21e45172',
    img: 'http://pngimg.com/uploads/guitar/guitar_PNG3369.png',
    name: 'Ziplin',
    price: 2000,
    quantity: 10,
    count: 1,
  },
  {
    id: '9fbec80e45172',
    img: 'http://pngimg.com/uploads/guitar/guitar_PNG3369.png',
    price: 20000,
    quantity: 10,
    name: 'TAKAMINE',
    count: 1,
  },
  {
    id: '9fbe00000',
    img: 'http://pngimg.com/uploads/guitar/guitar_PNG3369.png',
    name: 'Fender',
    price: 210000,
    quantity: 20,
    count: 4,
  },
  {
    id: '9fbe0000000007',
    img: 'http://pngimg.com/uploads/guitar/guitar_PNG3369.png',
    price: 2210000,
    quantity: 11,
    name: 'Test',
    count: 11,
  },
];
export {goods,basket};
