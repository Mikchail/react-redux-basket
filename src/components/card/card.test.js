import React from 'react';
import renderer from 'react-test-renderer'
import Card from './card';
import {goods} from '../../../mocks/goods';


const good = goods[1];

describe(`Test componet Card Snapshot`, () => {

  it(`should be Card`, () => {
    const tree = renderer.create(
      <Card good={good} addToBasket={()=>{}} hasInBasket={false} />
    ).toJSON();
   expect(tree).toMatchSnapshot();
  });
});