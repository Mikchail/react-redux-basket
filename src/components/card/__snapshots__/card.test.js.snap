// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test componet Card Snapshot should be Card 1`] = `
<li
  className="card"
>
  <h4
    className="card__title"
  >
    Fender
  </h4>
  <img
    alt="Fender"
    className="card__img"
    height="400"
    src="https://img95.699pic.com/element/40135/1104.png_860.png"
    width="300"
  />
  <p
    className="card__quantity"
  >
    <span
      className="span"
    >
      Quantity: 
    </span>
    2
  </p>
  <p
    className="card__price"
  >
    <span
      className="span"
    >
      Price: 
    </span>
    22000
  </p>
  <button
    className="button card__button"
    onClick={[Function]}
  >
    Add to basket
  </button>
</li>
`;
